### PR TITLE
some updates to download kraken db

### PR DESCRIPTION
**Pull request by chasemc**
I don't know which kraken db we're using so I just picked a random one and put it in the curl command (@Sidduppal )
Also @Sidduppal - take a look here to see if any kraken commands need to be adjusted to make it quicker/more optimal
https://ccb.jhu.edu/software/kraken/MANUAL.html#memory-usage-and-efficiency

See the last commit "comment out no-nworking" for code that was commented out to make the pipeline work. 

Notes: DIAMOND process doesn't exist (can take from Autometa?)

Still haven't actually checked pipeline because it fails because there's no `mmseqs2_db` channel yet

### CODE REVIEW

File: modules/local/kraken2.nf [hunk 1]
```diff
@@ -28,7 +27,7 @@ process KRAKEN2 {
         def prefix   = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
         """
         kraken2 \\
-            --db $db \\
+            --db ${params.kraken2_db} \\
```
Comment (WiscEvan) [inline modules/local/kraken2.nf:30]: Will this be mounted properly inside a docker container if not specified as a path to the process?

File: conf/test.config [hunk 2]
```diff
@@ -18,12 +18,9 @@ params {
     max_cpus   = 2
     max_memory = 6.GB
     max_time   = 6.h
+    kraken2_download_permission = true
+    ncbi_db = '/home/chase/Documents/large_db'
+    input = '/home/chase/Downloads/bench/input_csv.csv'
```
Comment (WiscEvan) [inline conf/test.config:23]: These paths will need to be updated

File: nextflow_schema.json [hunk 3]
```diff
@@ -45,30 +45,75 @@
...
                     "fa_icon": "fas fa-file-csv"
                 }
+            },
+            "required": [
+                "kraken2_db",
+                "ncbi_db"
+            ]
+        },
+        "autometa_va_params": {
+            "title": "autometa_va_params",
```
Comment (WiscEvan) [inline nextflow_schema.json:75]: What does the va stand for here?

File: nextflow_schema.json [hunk 4]
```diff
@@ -45,30 +45,75 @@
             }
         },
         "taxon_profiling_databases": {
-            "title": "Taxon-profiling Databases",
+            "title": "taxon_profiling_databases",
```
Comment (WiscEvan) [inline nextflow_schema.json:48]: Should this title be formatted for us or for the end-user?

**Review by WiscEvan — APPROVED** [review]
Nice 👍 Hopefully we can continue to get this cleaned up and functional soon. I had a few questions regarding some of the documentation in the schema. Otherwise, the main question I have is whether the kraken database will be properly mounted inside the docker container for the kraken process if provided as a parameter rather than specified as a path to the process. 

I think we can merge this in and forge ahead 🛠️ 👷

File: nextflow_schema.json (hunk 4 above)
Comment (chasemc) [reply nextflow_schema.json]: fixed, yeah I messed up the copy paste

File: nextflow_schema.json (hunk 3 above)
Comment (chasemc) [reply nextflow_schema.json]: fixed, typo, fixed, "v1"

File: conf/test.config (hunk 2 above)
Comment (chasemc) [reply conf/test.config]: fixed

File: modules/local/kraken2.nf (hunk 1 above)
Comment (chasemc) [reply modules/local/kraken2.nf:30]: fixed, I forgot about that

**Pull request merged**